app/controllers/users/sessions_controller.rb

```diff
@@ -167,14 +167,25 @@ def handle_valid_authentication
       UserAlerts::AlertUserAboutNewDevice.schedule_alert(event:) if new_device?
       EmailAddress.update_last_sign_in_at_on_user_id_and_email(
         user_id: current_user.id,
-        email: auth_params[:email],
+        email: last_email_from_sp,
       )
       user_session[:platform_authenticator_available] =
         params[:platform_authenticator_available] == 'true'
       check_password_compromised
       redirect_to next_url_after_valid_authentication
     end
 
+    def last_email_from_sp
+      return nil unless IdentityConfig.store.feature_select_email_to_share_enabled
```

Comment:
This is a change in behavior from how it worked previously, where we used `auth_params[:email]`. Would we expect that behavior to remain the same while the feature is disabled? Do we have test coverage for this?

Reply:
removed that in light of 
https://github.com/18F/identity-idp/pull/10951#discussion_r1715190798